code/go/0chain.net/smartcontract/storagesc/allocation_test.go

```diff
@@ -1654,9 +1654,13 @@ func TestStorageSmartContract_closeAllocation(t *testing.T) {
 	_, err = ssc.closeAllocation(&tx, alloc, balances)
 	requireErrMsg(t, err, errMsg1)
 
+	var conf = Config{
+		MaxChallengeCompletionTime: 30 * time.Minute,
+	}
+
```

Comment:
You don't need to create a new Config object. It was already created in [above](https://github.com/0chain/0chain/blob/d7321ef7a912eac8763e70eb6de034f69de6fadf/code/go/0chain.net/smartcontract/storagesc/allocation_test.go#L1643) in [createNewTestAllocation](https://github.com/0chain/0chain/blob/d7321ef7a912eac8763e70eb6de034f69de6fadf/code/go/0chain.net/smartcontract/storagesc/allocation_test.go#L1512).